tests/Soql/DriverErrorTest.php

```diff
@@ -18,12 +18,12 @@ public function it_should_retain_exception_instance() : void
         $expectedMessage   = 'The Driver Has Some Error';
         $expectedErrorCode = random_int(1, PHP_INT_MAX);
 
-        $stackedException = new SoqlError('Wrong Message', null, $expectedErrorCode);
+        $stackedException = new SoqlError(message: 'Wrong Message', errorCode: $expectedErrorCode);
 
         $exception = new DriverError($expectedMessage, $stackedException);
 
         self::assertSame($expectedMessage, $exception->getMessage());
         self::assertSame($expectedErrorCode, $exception->getErrorCode());
-        self::assertSame(0, $exception->getCode());
+        self::assertSame(expected: 0, actual: $exception->getCode());
```

Comment:
Why for this one and not the ones above?

Some of the usages of named parameters make sense but here I didn't get much the point.

Reply:
because a "magic number" is used.

Reply:
So what does `0` mean?